eventdata/challenges/large-shard-sizing.json

```diff
@@ -26,7 +26,7 @@
       "operation": {
         "operation-type": "create-index-template",
         "settings": {
-          "index.number_of_shards": {{ shard_count | default(1) }}
+          "index.number_of_shards": {{ number_of_shards | default(1) }}
```

Comment:
The change from the default is not documented. However, I'm ok if we do this in a follow-up PR.